ArcadiaParties.Data/Data/RoleSeed.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "Name": "Manager"
+  },
+  {
+    "Name": "Stager"
+  },
+  {
+    "Name": "Admin"
+  },
+  {
+    "Name": "Chelovek"
```

Comment:
Roles are not staff positions. For now, roles are "User" and "Administrator".

Reply:
All roles changed